test/e2e/inspect_test.go

```diff
@@ -126,6 +126,9 @@ spec:
 				"reconcile_info":  "",
 				"reconcile_state": "ok",
 			})
+
+			sortByKind(expected)
+			sortByKind(respRows)
```

Comment:
what prompted this change? is the diff summary not deterministic?

Reply:
While inspecting an app with the `--tree` flag. The result seemed to be deterministic and the order was generally, `Service, EndpointSlice, Endpoints` . The test initially had a different order but was passing erratically. Ordering seemed to be a viable option as we are checking for the existence of the resources and the test results were reproducible once the values were sorted in a deterministic manner.